client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -184,10 +185,18 @@ const Header: FunctionComponent = () => {
 				go();
 				return;
 			}
+
+			recordOnboardingComplete( {
+				isNewSite: !! newSite,
+				isNewUser: !! newUser,
+			} );
+
+			setIsRedirecting( true );
```

Comment:
the setIsRedirecting change is unrelated to analytics (I think) is it meant to go in?

Reply:
@roo2 yep, I added it in to prevent the analytics event [from firing twice](https://github.com/Automattic/wp-calypso/pull/41129#pullrequestreview-395241574) (it's the same pattern as used above when someone adds in a domain).